spec/Statements.tex

```diff
@@ -789,15 +789,22 @@ \section{The Use Statement}
 C1, C2, C3 have public module level definitions of the same symbol.
 
 An optional \sntx{limitation-clause} may be provided to limit the symbols made
-available by the use statement.  If an \chpl{except} list is provided, then all
+available by a use statement.  If an \chpl{except} list is provided, then all
```

Comment:
I was using "the" before to indicate the stronger connection between the limitation clause and the use itself.  I would be tempted to change this to "a particular use statement" instead as a result, but won't be heart-broken if you don't.

Reply:
It seems hard to believe someone would think it meant a different use statement, but as a compromise, how about "a given use statement"?

Reply:
That seems reasonable!